ksqldb-engine/src/main/java/io/confluent/ksql/internal/StorageUtilizationMetricsReporter.java

```diff
@@ -49,8 +51,9 @@
   private static final String METRIC_GROUP = "ksqldb_utilization";
 
   private Map<String, Map<String, TaskStorageMetric>> metricsSeen;
-  private Metrics metricRegistry;
+  private static Metrics metricRegistry;
```

Comment:
This should not be static - just leave it local to each member. We intentionally made it not-static to avoid re-using the instance across test runs.

Reply:
Is there a way to still reference it from a static context?

Reply:
I didn't change this because I'm not sure how to handle it, `configureShared` is called from a static context so it has to be static, meaning `getMaxTaskUsage` has to be static which calls `metricRegistry.metrics() which intellij is also telling me needs to be static to be called from this context. I can create a helper function to get `metricRegistry.metrics()` but it seems like it would have the same issue

Reply:
`configureShared` is passed a `MetricsRegistry` instance that you can register the metrics in.